source/refined-github.css

```
@@ -191,3 +191,9 @@ div[data-target='readme-toc.content'] div.blob-wrapper img {
 [data-selected-links^='repo_milestones ']:has([title='0']) svg {
 	margin-right: 20px; /* Push the text farther away from the text so it's cropped out */
 }
+
+/* Add override for CSS-in-JSS components that don't use regular .btn styles */
```

```suggestion
/* Add override for CSS-in-JS components that don't use regular .btn styles */
```

---

```suggestion
/* Add override for CSS-in-JSSS components that don't use regular .btn styles */
```

https://en.wikipedia.org/wiki/JavaScript_Style_Sheets

---

```suggestion
/* Add override for CS-in-JSS components that don't use regular .btn styles */
```

https://en.wikipedia.org/wiki/Computer_science

---

```suggestion
/* Add override for JAXA-in-ISS components that don't use regular .btn styles */
```

https://en.wikipedia.org/wiki/International_Space_Station